ignite/pkg/giturl/giturl.go

```diff
@@ -6,6 +6,11 @@ import (
 	"strings"
 )
 
+var (
```

Comment:
Seems like we don't use `pkg/giturl` anymore. Can we remove it altogether?

Reply:
There is but these people can stick to an older version of the CLI if they really want to use this package.

If we don't moderate/rm unused packages it could get out of control.